imagepicker/src/main/java/com/esafirm/imagepicker/helper/ExcludedMediaSingleton.java

```diff
@@ -0,0 +1,50 @@
+package com.esafirm.imagepicker.helper;
+
+import com.esafirm.imagepicker.model.Image;
+
+import java.io.File;
+import java.util.ArrayList;
+
+/**
+ * Code written by Qandeel Abbassi on 9/20/2018 at 7:03 PM.
+ */
+public class ExcludedMediaSingleton {
+    private static ExcludedMediaSingleton ourInstance;
+    private ArrayList<File> excludedImages;
+
+    private ExcludedMediaSingleton() {
+    }
+
+    public static ExcludedMediaSingleton getInstance() {
+        if (ourInstance == null) { //if there is no instance available... create new one
+            ourInstance = new ExcludedMediaSingleton();
+        }
+        return ourInstance;
+    }
+
+    public void resetExclusions() {
+        if (excludedImages != null) {
+            excludedImages.clear();
```

Comment:
I think this could lead to some unexpected behavior since `excludedImages` is shared between ImagePicker and others screen on the app. 

If you want to do this, please copy the list instead of just assigning it in `setExcludedImageFiles()`